aptos-move/e2e-benchmark/data/calibration_values.tsv

```diff
@@ -24,6 +24,8 @@ FungibleAssetMint	34	0.934	1.063	236.7
 IncGlobalMilestoneAggV2 { milestone_every: 1 }	34	0.913	1.040	33.7
 IncGlobalMilestoneAggV2 { milestone_every: 2 }	34	0.915	1.050	19.1
 EmitEvents { count: 1000 }	34	0.946	1.091	8887.0
+APTTransferWithPermissionedSigner	34	0.946	1.091	1068.0
+APTTransferWithMasterSigner	34	0.946	1.091	185.0
```

Comment:
so the permission signer is 5x more expensive than master signer

Reply:
Wait that's actually much huger than I would thought. Is it actually due to the switch to the new big table?

Reply:
this is the move code alone, not the whole transaction itself.

it shouldn't be from the BigMap - it has similar perf to SmartTable

Reply:
Oh I see. So this is not the txn e2e speed. That would make sense.